packages/oauth-providers/package.json

```diff
@@ -2,96 +2,38 @@
   "name": "@hono/oauth-providers",
   "version": "0.7.0",
   "description": "Social login for Hono JS, integrate authentication with facebook, github, google and linkedin to your projects.",
+  "type": "module",
   "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
   "files": [
     "./dist"
   ],
   "scripts": {
-    "test": "vitest",
-    "build": "tsup && publint",
-    "watch": "tsup --watch",
-    "publint": "publint",
-    "release": "yarn build && yarn test && yarn publint && yarn publish"
+    "build": "tsup ./src/index.ts ./src/providers/**/index.ts ./src/providers/**/types.ts --no-splitting",
+    "prepack": "yarn build",
+    "publint": "attw --pack && publint",
+    "test": "vitest"
   },
   "exports": {
     ".": {
       "import": {
-        "types": "./dist/index.d.mts",
-        "default": "./dist/index.mjs"
-      },
-      "require": {
         "types": "./dist/index.d.ts",
         "default": "./dist/index.js"
-      }
-    },
-    "./google": {
-      "import": {
-        "types": "./dist/providers/google/index.d.mts",
-        "default": "./dist/providers/google/index.mjs"
-      },
-      "require": {
-        "types": "./dist/providers/google/index.d.ts",
-        "default": "./dist/providers/google/index.js"
-      }
-    },
-    "./facebook": {
-      "import": {
-        "types": "./dist/providers/facebook/index.d.mts",
-        "default": "./dist/providers/facebook/index.mjs"
-      },
-      "require": {
-        "types": "./dist/providers/facebook/index.d.ts",
-        "default": "./dist/providers/facebook/index.js"
-      }
-    },
-    "./github": {
-      "import": {
-        "types": "./dist/providers/github/index.d.mts",
-        "default": "./dist/providers/github/index.mjs"
-      },
-      "require": {
-        "types": "./dist/providers/github/index.d.ts",
-        "default": "./dist/providers/github/index.js"
-      }
-    },
-    "./linkedin": {
-      "import": {
-        "types": "./dist/providers/linkedin/index.d.mts",
-        "default": "./dist/providers/linkedin/index.mjs"
-      },
-      "require": {
-        "types": "./dist/providers/linkedin/index.d.ts",
-        "default": "./dist/providers/linkedin/index.js"
-      }
-    },
-    "./x": {
-      "import": {
-        "types": "./dist/providers/x/index.d.mts",
-        "default": "./dist/providers/x/index.mjs"
-      },
-      "require": {
-        "types": "./dist/providers/x/index.d.ts",
-        "default": "./dist/providers/x/index.js"
-      }
-    },
-    "./discord": {
-      "import": {
-        "types": "./dist/providers/discord/index.d.mts",
-        "default": "./dist/providers/discord/index.mjs"
       },
       "require": {
-        "types": "./dist/providers/discord/index.d.ts",
-        "default": "./dist/providers/discord/index.js"
+        "types": "./dist/index.d.cts",
+        "default": "./dist/index.cjs"
       }
     },
-    "./twitch": {
+    "./*": {
```

Comment:
I also reduced all the seperate providers `exports` into a single [subpath pattern](https://nodejs.org/docs/latest-v18.x/api/packages.html#subpath-patterns), hopefully that makes it a little easier to add new providers in the future

Reply:
I've confirmed that this works well. Thanks!